app/src/main/java/org/imozerov/streetartview/ui/explore/ExploreArtActivity.kt

```diff
@@ -14,31 +15,46 @@ import android.support.v7.widget.SearchView
 import android.util.Log
 import android.view.View
 import android.view.inputmethod.InputMethodManager
+import android.widget.LinearLayout
 import com.jakewharton.rxbinding.support.v7.widget.RxSearchView
 import kotlinx.android.synthetic.main.activity_explore_art.*
+import kotlinx.android.synthetic.main.bottom_details.*
 import org.imozerov.streetartview.R
+import org.imozerov.streetartview.StreetArtViewApp
+import org.imozerov.streetartview.storage.IDataSource
 import org.imozerov.streetartview.ui.detail.DetailArtObjectActivity
 import org.imozerov.streetartview.ui.detail.interfaces.ArtObjectDetailOpener
+import org.imozerov.streetartview.ui.explore.interfaces.ArtObjectDigestOpener
 import org.imozerov.streetartview.ui.explore.interfaces.Filterable
 import org.imozerov.streetartview.ui.explore.list.ArtListFragment
 import org.imozerov.streetartview.ui.explore.map.ArtMapFragment
+import org.imozerov.streetartview.ui.extensions.loadImage
+import org.imozerov.streetartview.ui.model.ArtObjectUi
 import rx.android.schedulers.AndroidSchedulers
 import rx.subscriptions.CompositeSubscription
 import java.util.*
+import javax.inject.Inject
+
+class ExploreArtActivity : AppCompatActivity(), ArtObjectDetailOpener, ArtObjectDigestOpener {
 
-class ExploreArtActivity : AppCompatActivity(), ArtObjectDetailOpener {
     val TAG = "ExploreArtActivity"
 
     val compositeSubscription = CompositeSubscription()
+    var bottomSheetBehavior: BottomSheetBehavior<LinearLayout>? = null
```

Comment:
If I understand right, the bottom sheet logic affects map only (not gallery). So, why is it in the activity, not in the corresponding fragment?